pkg/whitesource/configHelper.go

```diff
@@ -47,6 +50,18 @@ func (s *ScanOptions) RewriteUAConfigurationFile(utils Utils, projectName string
 	newConfigMap := cOptions.updateConfig(&uaConfigMap)
 	newConfig := properties.LoadMap(newConfigMap)
 
+	if verbose {
+		var printLog string
+		for _, p := range parametersForLog {
```

Comment:
@akram8008 If buildTool is configured as docker, by default while printing the wss-unified-agent.config contents we could be able to see these parameters. 

Do we have any use case to print it while configured with other buildTool also ? In that case the parameter "excludeBaseImage" and "dockerfilePath" will not be available by default in the uaContent.

Reply:
If buildTool is not set to 'docker', then these parameters will be displayed such:
**_debug whitesourceExecuteScan - ScanUA configuration: docker.excludeBaseImage:null docker.dockerfilePath:null_**